src/components/SelectColors/SelectColors.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * CSS class of the selectColors.
+   */
+  className: PropTypes.string,
+
+  /**
+   * Array of values to use in the selectColors element.
+   */
+  options: PropTypes.arrayOf(
+    PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+      PropTypes.shape({
+        value: PropTypes.oneOfType([
+          PropTypes.string,
+          PropTypes.number,
+          PropTypes.arrayOf(
+            PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+          ),
+        ]).isRequired,
+        label: PropTypes.string.isRequired,
+      }),
+    ]),
+  ),
+
+  /**
+   * Function triggered on select's change event.
+   * @param {Event} event The change event object.
+   */
+  onChange: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  options: undefined,
+  className: 'tm-selectcolors',
+};
+
+/**
+ * This component displays a simple selectColors HTML element.
+ */
+const SelectColors = function(props) {
+  const { options } = props;
+  const { className } = props;
+  const { onChange } = props;
+  if (!options) {
+    return null;
+  }
+
+  return React.createElement(
```

Comment:
We use https://fr.reactjs.org/docs/introducing-jsx.html

Reply:
Nepalese version of the doc is not yet ready, here is the english one :)
https://en.reactjs.org/docs/introducing-jsx.html

Reply:
You can check other functional components example in the lib, and follow the same pattern, such as:
https://github.com/geops/react-spatial/blob/master/src/components/Select/Select.js

Reply:
Done